openedx/core/djangolib/waffle_utils.py

```diff
@@ -82,11 +78,20 @@ def override_for_request(self, switch_name, active=True):
         """
         namespaced_switch_name = self._namespaced_setting_name(switch_name)
         self._cached_switches[namespaced_switch_name] = active
-        print "Set cached value"
-        print self._get_request_cache()
-        print "\n"
         log.info(u"%sSwitch '%s' set to %s for request.", self.log_prefix, namespaced_switch_name, active)
 
+    @contextmanager
+    def override_in_model(self, switch_name, active=True):
```

Comment:
I added this method that allows overriding Waffle flags at the model level in addition to the request cache.